EDA/eda.ipynb

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
<p>Stopword removal and word frequency plots are interesting but they are not necessarily giving us they key topics or the themes these bills correspond to. For example we are not getting much information from may, shall, hereby, department etc  It is worth doing a topic and entity detection here to identify the key areas these bills talk about and then plotting the top topics or entities. You could use a pre trained model or a library to do these. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-maple-bill-summarization/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Agreed, here we just tried to explore as many things about our dataset as possible, and we agree that this isn't very insightful. We also ran a Top2Vec model on our bills, and extracted some more meaningful information. </p>